drivers/nuttxcan/nuttxcan.c

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2016 UAVCAN Team
+ *
+ * Distributed under the MIT License, available in the file LICENSE.
+ *
+ */
+
+#include "nuttxcan.h"
+#include <fcntl.h>
+#include <poll.h>
+#include <string.h>
+#include <unistd.h>
+#include <nuttx/can.h>
+
+/**
+ * Initializes the NuttXCAN instance.
+ */
```

Comment:
I strongly suggest to move all documentation comments into the header file. The main library partially deviates from this convention, but it should be fixed someday.